deployment/helm/README.md

```diff
@@ -189,6 +214,27 @@ curl -iv --resolve edgex:443:<MetalLB IP> --cacert server-ca.pem --cert client.p
 {"apiVersion":"v2","timestamp":"Wed Feb  2 18:32:57 UTC 2022"}
 ```
 
+*** 
+TODO get an JWT from Helm
+***
```

Comment:
details to be filled in?

Reply:
This was a big hole.  I had to make a lot of changes.  No wonder I hadn't submitted PR review earlier.  This has been updated.